### PR TITLE
Eucalyptus.2 release notes

### PR DESCRIPTION
**Pull request by nedbat**

### CODE REVIEW

**Comment by nedbat** [comment]
@edx/doc OK?

**Comment by srpearce** [comment]
@nedbat I'm looking at the Dogwood release notes for comparison, and it looks like all the items under patch releases are complete sentences—I can make suggestions for the items in this PR if you'd like.

File: en_us/open_edx_release_notes/source/eucalyptus.rst [hunk 1]
```diff
@@ -550,6 +550,26 @@ account.
...
+**************
+
+==============================
+2 September 2016: Eucalyptus.2
+==============================
+
+* Fixed a problem with django-debug-toolbar raising an exception: "process() 
+  takes exactly 3 arguments (2 given)."
+
+* Fixed poor discussion forum performance. (:jira:`TNL-5173`)
```
Comment (srpearce) [inline en_us/open_edx_release_notes/source/eucalyptus.rst]: Course discussion performance has been improved.

File: en_us/open_edx_release_notes/source/eucalyptus.rst [hunk 2]
```diff
@@ -550,6 +550,26 @@ account.
...
+  
+* Fixed a problem preventing adding a comment to response on inline
+  discussions. (:jira:`TNL-5389`)
```
Comment (srpearce) [inline en_us/open_edx_release_notes/source/eucalyptus.rst]: Learners can now correctly add a comment to a response in inline course discussions.

File: en_us/open_edx_release_notes/source/eucalyptus.rst [hunk 3]
```diff
@@ -550,6 +550,26 @@ account.
...
+
+* Fixed links to vertical blocks. (:jira:`TNL-5003`)
```
Comment (srpearce) [inline en_us/open_edx_release_notes/source/eucalyptus.rst]: Links to vertical blocks have been fixed.

File: en_us/open_edx_release_notes/source/eucalyptus.rst [hunk 4]
```diff
@@ -550,6 +550,26 @@ account.
...
+**************
+Patch Releases
+**************
+
+==============================
+2 September 2016: Eucalyptus.2
+==============================
+
+* Fixed a problem with django-debug-toolbar raising an exception: "process() 
+  takes exactly 3 arguments (2 given)."
```
Comment (srpearce) [inline en_us/open_edx_release_notes/source/eucalyptus.rst:566]: A problem that caused the `django-debug-toolbar` element(?) to raise a "process() takes exactly 3 arguments (2 given)" exception error has been fixed.

**Pull request merged**